src/XrdCeph/XrdCephOss.cc

```diff
@@ -41,6 +41,8 @@
 #include "XrdCeph/XrdCephOssDir.hh"
 #include "XrdCeph/XrdCephOssFile.hh"
 
+#include <chrono>
```

Comment:
Different pull request?

Reply:
This include is not necessary if `xrootd-ceph` is build separately from xrootd itself.

Reply:
So yes, if it stays it should go to a separate PR.

Reply:
include removed.